main.ts

```diff
@@ -13,7 +16,10 @@ app.use(express.json());
 
 app.post("/users", getUser)
 app.post("/users/:userId/recipes", addRecipe)
+app.get('/users/:userId/ingredients', getAllIngredients)
+app.get('/users/:userId/ingredients/:ingredientId', getSingleIngredient)
 app.get('/users/:userId/recipes', getAllUserRecipes)
 app.get('/users/:userId/recipes/:recipeId', getRecipeById)
+app.post('/users/:userId/ingredient', addIngredient)
```

Comment:
```suggestion
app.post('/users/:userId/ingredients', addIngredient)
```

REST APIs have plural route names. You guys need to get on top of this single/plural thing.

Reply:
done

Reply:
is this updated / accurate to the API documentation too?

Reply:
yup, done